src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/package.scala

```diff
@@ -14,15 +15,21 @@ package object config {
   }
 
   implicit val dataprocConfigReader: ValueReader[DataprocConfig] = ValueReader.relative { config =>
-    DataprocConfig(config.getString("applicationName"),
+    DataprocConfig(
+      config.getString("applicationName"),
       config.getString("serviceAccount"),
       config.getString("dataprocDefaultRegion"),
       config.getString("leoGoogleBucket"),
       config.getString("dataprocDockerImage"),
       config.getString("jupyterProxyDockerImage"),
-      config.getString("clusterFirewallRuleName"),
-      config.getString("clusterFirewallVPCNetwork"),
-      config.getString("clusterNetworkTag"),
+      config.getString("includeDeletedKey"),
```

Comment:
It seems weird to make `includeDeletedKey` a config -- can't it just be a constant in code?

Reply:
Yeah, I can remove it from the config